proof/crefine/lib/Corres_C.thy

```diff
@@ -988,14 +988,10 @@ lemma ccorres_mapM_x_while_gen':
   unfolding mapM_x_def
   apply (rule ccorres_rel_imp)
    apply (rule ccorres_sequence_x_while_gen'[where xf_update=xf_update])
-        apply (clarsimp simp only: length_map nth_map rl)
-       apply (simp add: assms hi[simplified])+
+         apply (clarsimp simp only: length_map nth_map rl)
+        apply (simp add: assms hi[simplified])+
   done
 
-lemmas ccorres_mapM_x_while'
-    = ccorres_mapM_x_while_gen' [OF _ _ _ _ _ i_xf_for_sequence, folded word_bits_def,
-                                 where j=1, simplified]
```

Comment:
there's a bunch of these getting removed... are they unused, already proved, bad ideas?

Reply:
oh never mind, they got moved to the context below